android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -644,11 +681,18 @@ protected void onDetachedFromWindow() {
                 BraveActivity.getBraveActivity().setNewsItemsFeedCards(mNewsItemsFeedCard);
             }
 
-            if (mSettingsBar != null && mCompositorView != null) {
-                if (mSettingsBar.getId() == R.id.news_settings_bar) {
-                    mSettingsBar.setVisibility(View.GONE);
-                    mSettingsBar.setAlpha(0f);
-                }
+            Tab tab = BraveActivity.getBraveActivity().getActivityTab();
```

Comment:
there are no checks on `if (BraveActivity.getBraveActivity() != null) {` but I see them in other places. Should we check it here as well?

Reply:
@SergeyZhukovsky i removed the entire "Tab" block. leftover